### ISSUE TITLE
Timeout option

### ISSUE THREAD

**Issue by deStrO** [driesvints/vat-calculator#36]
Hello,

Yesterday the VIES service was down and I had my website blocked on loading when I was using the class VatCalculator. It appear that the SoapClient was in kind of timeout as the VIES service was down.

Any luck to add a timeout param?

**Comment by nicolasbeauvais** [comment]
Same thing since this morning:
![screenshot-2017-01-23_14-53-38](https://cloud.githubusercontent.com/assets/2951704/22206499/dc214ad6-e17b-11e6-8c8d-eefc907cf950.png)

I tried to add a `connection_timeout` parameter to the `SoapClient` constructor but it had no effect:
```php
$this->soapClient = new SoapClient(self::VAT_SERVICE_URL, ['connection_timeout' => 5]);
```

@mpociot  Any idea of a workaround for this?

Thanks!

**Comment by AlinGhinoiu** [comment]
Same thing yesterday.

**Comment by kentdahl** [comment]
Ditto today - http://sorry.ec.europa.eu/

> The EUROPA server is temporarily unavailable
We apologise for any inconvenience this may cause you.

**Comment by driesvints** [comment]
Closing this issue because it's inactive, already solved, old or not relevant anymore. Feel to open up a new issue if you're still experiencing this.

**Comment by pmochine** [comment]
Unfortunately, I had the same problem yesterday. Does the `"connection_timeout"` code work?
I ran my PHPUnit test and it took: `Time: 04:19.413` to finish it, but my tests failed because VIES was down. For comparison, my test now takes about `Time: 00:19.873`. 

Obviously I don't want my user to wait 30seconds or more until I throw out an error 🙈
